src/plugins/performance-plugin.ts

```diff
@@ -1,21 +1,25 @@
-import Perfume from 'perfume.js';
 import perfumePlugin from '@analytics/perfumejs';
 
 // Track performance metrics
-const performancePlugin = ({ sendTo }: { sendTo?: string[]}) => {
-  const destinations = Array.isArray(sendTo)
-    ? sendTo.reduce((acc, tracker) => ({ ...acc, [tracker]: true }), {})
-    : { all: true };
-  return perfumePlugin({
-    perfume: Perfume,
-    category: 'perfMetrics',
-    destinations,
-    perfumeOptions: {
-      resourceTiming: true,
-      elementTiming: true,
-      maxMeasureTime: 15000,
-    },
-  });
+const performancePlugin = ({ sendTo }: { sendTo?: string[] }) => {
+  if (typeof window !== 'undefined') {
+    const Perfume = import('perfume.js');
```

Comment:
I believe this will return a `Promise`, right?

Reply:
We will need to change `performancePlugin` to be `async` so we can `await import('perfume.js');`

Reply:
Ah yeah great shout, I was playing around with async await on the imports and updated the syntax but it's promised based. Will update now 👌